genesyscloud/scripts/resource_scripts_schema.go

```diff
@@ -85,5 +85,8 @@ func ExporterScript() *resourceExporter.ResourceExporter {
 			RetrieveAndWriteFilesFunc: ScriptResolver,
 			SubDirectory:              "scripts",
 		},
+		DataSourceResolver: map[*resourceExporter.DataAttr]*resourceExporter.ResourceAttr{
```

Comment:
Can you remind me of what the `DataSourceResolver` is and what the difference is between this and `ExportAsData`? There's not really any good comments in the definition to help remind us of the distinction between the two.